### PR TITLE
Example notebook for using Plottr with Qcodes measurements for live plotting

### PR DESCRIPTION
**Pull request by Akshita07**
Changes proposed in this pull request:
- Adds an example notebook to show how plottr can be used with qcodes measurements for live plotting

To Do:
- [x] Add a gif to show live plotting

### CODE REVIEW

File: docs/examples/plotting/How-to-use-Plottr-with-QCoDeS-for-live-plotting.ipynb [hunk 1]
```diff
@@ -0,0 +1,206 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to use Plottr with QCoDeS for live plotting?\n",
+    "\n",
+    "Note: This notebook assumes that the user has a `conda` environment on their system with latest versions of `plottr` installed. If you do not have plottr installed, please use command `pip install plottr` for installing it into your environment.\n",
```
Comment (astafan8) [inline docs/examples/plotting/How-to-use-Plottr-with-QCoDeS-for-live-plotting.ipynb:9]: perhaps rather refer to plottr's readme (or docs?) for installation instructions becasue qcodes shouldnb't need to maintain those

File: docs/examples/plotting/How-to-use-Plottr-with-QCoDeS-for-live-plotting.ipynb [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+    "                             (dmm.v1, get_v))\n",
+    "        \n",
+    "    dataset = datasaver.dataset\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "On starting this measurement, a plot window (as shown below) will open automatically. This plot will keep refreshing at the interval rate set by you till the measurement runs.\n",
```
Comment (astafan8) [inline docs/examples/plotting/How-to-use-Plottr-with-QCoDeS-for-live-plotting.ipynb:170]: and i think this nice exampke notebook can be concluded with somthling like "refer to plottr docs and whatever for more information on it's features"

File: docs/examples/plotting/How-to-use-Plottr-with-QCoDeS-for-live-plotting.ipynb [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+   "source": [
+    "\n",
+    "meas = Measurement(exp=exp)\n",
+    "meas.register_parameter(dac.ch1)\n",
+    "meas.register_parameter(dmm.v1, setpoints=(dac.ch1,))\n",
+    "\n",
+    "meas.write_period = 0.1\n",
+    "\n",
+    "\n",
+    "with meas.run() as datasaver:\n",
```
Comment (astafan8) [inline docs/examples/plotting/How-to-use-Plottr-with-QCoDeS-for-live-plotting.ipynb:155]: this is great, but perhaps an extra example with one of the doNd functions would be also cool.

File: docs/examples/plotting/How-to-use-Plottr-with-QCoDeS-for-live-plotting.ipynb [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now open an anaconda prompt and activate your conda environment where plottr is installed. Start Inspectr GUI with `plottr-inspectr` command. You will see a window as show below. In this window, set `Refresh interval (s)` to the desired value and check `Auto-plot new`. Using `File` button load the database initialized (or created) above.\n",
+    "\n",
+    "<div>\n",
+    "<img src=\"files/plottr.jpg\" width=\"400\" height=\"400\"/>\n",
```
Comment (astafan8) [inline docs/examples/plotting/How-to-use-Plottr-with-QCoDeS-for-live-plotting.ipynb:118]: you could choose to store the static files one level up, at the `docs/examples/files` directiory, for consistency (?)

**Review by astafan8 — COMMENTED** [review]
looks nice and sweet!

please make sure that  the built docs correctly include the images :)

File: docs/examples/plotting/How-to-use-Plottr-with-QCoDeS-for-live-plotting.ipynb [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+    "initialise_or_create_database_at(db_file_path)\n",
+    "exp = load_or_create_experiment(experiment_name='plottr_for_live_plotting_with_subsecond_refresh_rate',\n",
+    "                          sample_name=\"no sample\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now open an anaconda prompt and activate your conda environment where plottr is installed. Start Inspectr GUI with `plottr-inspectr` command. You will see a window as show below. In this window, set `Refresh interval (s)` to the desired value and check `Auto-plot new`. Using `File` button load the database initialized (or created) above.\n",
```
Comment (FarBo) [inline docs/examples/plotting/How-to-use-Plottr-with-QCoDeS-for-live-plotting.ipynb:115]: You can say, it is possible to drag and drop the database file into the plottr

File: docs/examples/plotting/How-to-use-Plottr-with-QCoDeS-for-live-plotting.ipynb (hunk 1 above)
Comment (Akshita07) [reply docs/examples/plotting/How-to-use-Plottr-with-QCoDeS-for-live-plotting.ipynb:9]: Sure, will do.

**Comment by jenshnielsen** [comment]
I think it would be good to link to this notebook from the 15 min to qcodes intro notebook

File: docs/examples/15_minutes_to_QCoDeS.ipynb [hunk 6]
```diff
@@ -1325,6 +1325,20 @@
...
+   "metadata": {},
+   "source": [
+    "### QCoDeS measurements live plotting with Plottr"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Plottr supports and is recommended for QCoDeS measurements live plotting. [How to use plottr with QCoDeS for live plotting](http://qcodes.github.io/Qcodes/examples/plotting/How-to-use-Plottr-with-QCoDeS-for-live-plotting.html) notebook contains more information."
```
Comment (jenshnielsen) [inline docs/examples/15_minutes_to_QCoDeS.ipynb:1340]: Please use a relative link for this. Like this https://nbsphinx.readthedocs.io/en/0.8.0/markdown-cells.html#Links-to-Other-Notebooks

**Comment by jenshnielsen** [comment]
ready to land once the gif is added

**Comment by astafan8** [comment]
ready to land once a **smaller** gif is added :)

**Comment by FarBo** [comment]
> ready to land once a smaller gif is added :)

What is the small gif?

**Comment by jenshnielsen** [comment]
merge via squash to avoid including the old git

**Review by jenshnielsen — APPROVED** [review]

**Pull request merged**